### PR TITLE
docs(icon): documentation updates

### PR DESCRIPTION
**Pull request by pixelflips**
# Description

Updates documentation of icon component

Fixes https://kajabi.atlassian.net/browse/DSS-1182

## Type of change

- [x] This change is a documentation update

# How Has This Been Tested?

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [x] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

### CODE REVIEW

File: libs/core/src/components/pds-icon/docs/pds-icon.mdx [hunk 1]
```diff
@@ -1,67 +1,79 @@
...
+    <pds-icon name="bulb" color="orange"></pds-icon>
+    <pds-icon name="bulb" color="green"></pds-icon>
+    <pds-icon name="bulb" color="blue"></pds-icon>
   </>
 </DocCanvas>
 
-It also accepts any custom size attribute.
+### Size
+
+The `size` property also accepts custom sizes.
```
Comment (ju-Skinner) [inline libs/core/src/components/pds-icon/docs/pds-icon.mdx]: We should probably include what t-shirt sizes are available.

below is what was taken from code. 
```
| 'small'   // 12px
| 'regular'  // 16px
| 'medium'  // 20px
| 'large'   // 24px
```

File: libs/core/src/components/pds-icon/docs/pds-icon.mdx [hunk 2]
```diff
@@ -1,67 +1,79 @@
...
-The `color` property allows the icon to render in the specified color. This can be either string or hex format.
+### Color
+
+The `color` property allows the icon to be displayed in a specified color. Accepted formats include:
+
+- String: `red`
+- HEX:  `#FF0000`
+- RGB:  `rgb(255, 0, 0)`
+- HSL: `hsl(0, 100%, 50%)`
+- Pine CSS variables: `var(--pine-color-red-500)`
```
Comment (ju-Skinner) [inline libs/core/src/components/pds-icon/docs/pds-icon.mdx:33]: I confirmed this works. Should we include these in the examples below?

**Review by ju-Skinner — APPROVED** [review]
I left two comments, not blocking.

File: libs/core/src/components/pds-icon/docs/pds-icon.mdx (hunk 1 above)
Comment (pixelflips) [reply libs/core/src/components/pds-icon/docs/pds-icon.mdx]: Added. Good call.

File: libs/core/src/components/pds-icon/docs/pds-icon.mdx (hunk 2 above)
Comment (pixelflips) [reply libs/core/src/components/pds-icon/docs/pds-icon.mdx:33]: Good call. Added each to the examples.

**Review by QuintonJason — APPROVED** [review]

**Pull request merged**